features/step_definitions/question_steps.rb

```diff
@@ -2,15 +2,12 @@
   visit '/'
   click_link 'Create new form'
 
-  within '[name="first-researcher"]' do
-    fill_in 'Full name', with: 'Rachel Researcher'
-    fill_in 'Telephone number', with: '012345678'
-    fill_in 'Email', with: 'rachel@researcher.com'
-  end
+  fill_in 'researcher_name', with: 'Rachel Researcher'
+  fill_in 'researcher_phone', with: '012345678'
+  fill_in 'researcher_email', with: 'rachel@researcher.com'
+
+  fill_in 'researcher_other_name', with: 'Steve Secondresearcher'
```

Comment:
Can we stick to filling in fields by labels? There are two good reasons for doing this:

1. It shows that our labels are consistently tied to their fields (this was very useful indeed in `move-to-form-with` – I could see what linkages I'd broken that weren't covered in `spec`)
2. There's a big conflict coming down the pipe from that very branch :wink:

I can't see the reason for changing this unless it's "well I changed a few labels and I don't want to change this test again", which is on the face of it a good reason but for 1. above.

Reply:
We can't use the label because there are two fields on the screen with 'first name'

Reply:
There are two "Full name" fields, yes, and we can because we have (or at least had) the `within` blocks

Reply:
(you're quite right to say we can't go by label alone; we previously had to reference the fieldset by name in a `within`, is what I'm trying to say)